### PR TITLE
Added security and privacy note for add-ons to the user guide

### PR DESCRIPTION
**Pull request by Adriani90**
### Link to issue number:
n/a
### Summary of the issue:
In many discussions, especially in corporate environments but also when users of other screen readers change to NVDA, there is no common perception of security and privacy status when using add-ons throughout the community of NVDA users.
### Description of user facing changes
Users will get a common sense for the perception of the status of security and privacy when using add-ons.
### Description of development approach
Discussion #16241 provides more details and current developments.

### Testing strategy:
Tested that the formating of the text appears correctly in the user guide, including the link to the community review section.

### Known issues with pull request:
None

### Code Review Checklist:

- [x] Documentation:
  - User Documentation
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [ ] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -2925,6 +2926,32 @@ If you install an add-on with paid components and change your mind about using i
 The Add-on Store is accessed from the Tools submenu of the NVDA menu.
 To access the Add-on Store from anywhere, assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+++ Note on security and privacy when using Add-ons ++[AddonSecurityandPrivacy]
+Installing add-ons in NVDA leads to integration of external code into NVDA's functionality in order to enhance NVDA or make new features possible.
+Add-ons can also use external libraries and third party services to serve the purpose and provide the features for which they have been developed.
+Add-ons can be developed by every person or company, and the review process for these external feature providers happens when they are submitted to the NVDA’s official add-on store.
+
+The review process of add-ons is still in development, so most of add-ons are not officially reviewed yet.
```
Comment (lukaszgo1) [inline user_docs/en/userGuide.t2t:2934]: 'officially' here implies 'by NV Access', which was never the case and probably never would.

File: user_docs/en/userGuide.t2t [hunk 2]
```diff
@@ -2925,6 +2926,32 @@ If you install an add-on with paid components and change your mind about using i
...
+
+The review process of add-ons is still in development, so most of add-ons are not officially reviewed yet.
+However, many add-ons have discussions areas where users can exchange feedback. The [community review area #AddonStoreReviews] can be accessed via the actions menu of the add-on.
+
+Installed Add-ons or extensions (not only in NVDA) might in general introduce security and/or privacy vulnerabilities, depending on the permissions they need and actions they perform in order to provide the desired functionality.
+Risks can be e.g.
+- Insecure network connections
+- Files stored with insecure file permissions or in an unprotected location
+- Sensitive information written to an easily available log file
+- Web browser vulnerabilities
```
Comment (lukaszgo1) [inline user_docs/en/userGuide.t2t:2942]: This risk seems irrelevant to NVDA add-ons.

**Review by lukaszgo1 — CHANGES_REQUESTED** [review]
Not related to the content of the PR, but please don't submit PR's from the master branch of your fork. This will inevitably introduce difficulties for you later on.

**Comment by Adriani90** [comment]
Closing this one in favor of #16316.

**Pull request closed without merging**